src/octopeer-github/content/ElementEventBinding/ElementEventBindingData.ts

```diff
@@ -0,0 +1,44 @@
+/// <reference path="../Types/EventID.ts"/>
+/**
+ * Created by Mathias on 02-06-2016.
+ * This interface enforces the data elements of the ElementEventBindings.
+ * eventID is the corresponding ID given in the eventID class.
+ * name is the descriptive string that belongs to the event.
+ */
+interface ElementEventBindingData {
+    eventID: EventID;
+    hookToDOM?: (elementSelectionBehaviour: GenericElementSelectionBehaviour) => void;
+    name: string;
+}
+
+/**
+ * elementEventBindingDataList contains all data about the different ElementEventBindings.
+ * @type {{eventID: EventID, name: string}[]}
+ */
+// tslint:disable-next-line:no-unused-variable
+const elementEventBindingDataList: ElementEventBindingData[] = [
```

Comment:
Please be consistent in your naming. I recall that this list for the ESBs was called `unsortedElementSelectionBehaviourData`, that doesn't quite match this one.